### ISSUE TITLE
Turning Auto Velocity "ON" without properly setting it sets it to a weird value

### ISSUE THREAD

**Issue by martinkf** [raveitoutofficial/raveitout#35]
Weird bug to explain

Steps to reproduce:
-Launch the game as normal;
-When reaching the song select screen, select a song;
-When selecting a difficulty, access the Option List;
-Note that Auto Velocity is OFF;
-Go into the Auto Velocity sub-menu and set it to ON without inputting any value;
-Leave the Option List;
-Note that the Auto Velocity value will have set to 200 and will weirdly decrease little by little over time (the value in the song info box can be updated by launching the Option List again)

Additionally, at this stage, modifying the value (by using the "AV +100", "AV -10", "AV +10", "AV -100" Option List buttons) will have no effect over the value - the only way to restore the Auto Velocity feature is turning it OFF and then setting a value straight away, without touching the ON button.

So basically, the whole idea of having a "ON" button over there is kinda weird and its not actually serving any purpose, as a player that wants AV turned on will have to input some value anyway

**Comment by RhythmLunatic** [comment]
The feature was never finished because of limitations, so wait for #39.
